src/core/cgroup.c

```diff
@@ -7,6 +7,9 @@
 #include "alloc-util.h"
 #include "blockdev-util.h"
 #include "bpf-devices.h"
+#if HAVE_LIBBPF
+#include "bpf-lsm.h"
+#endif
```

Comment:
could we move this ifdeffery into the header file please? it's usually better to hide ifdeffery in provider not the user of the code if possible